### DISCUSSION TITLE
UX Improvement

### DISCUSSION THREAD

**Discussion by sonna-io** [dailydotdev/daily#573] [Feature Requests]
Hi there!
daily.dev is an amazing application that helps saving a lot of time by bringing to devs exactly the resources they need to rise to the top.
However, it would be super interesting if we were able to  directly open and render the feed items links inside the application. Basically, render the target article directly in the app. Opening each article in a new tab is cumbersome and when we choose to open the article in the current tab, we suffer a loss of our feed because when we navigate backwards to the daily.dev homepage, the app load content again, then new articles appear and the old ones that may have also interested us disappear. 

This is what I mean :
![likeThis](https://user-images.githubusercontent.com/47452191/172588280-3733945d-9a82-4d9f-9586-ac12faf25c03.png)

Thanks a lot for this awesome app !

-------------------------------------------------------------------------
Edit : 
While waiting for the publishers' sites to come into the application, it is the application that goes to the publishers' sites !
Just Insane 🤯
Game changing move 👏👏
Thanks again great mind @idoshamun

![Screenshot (2)](https://user-images.githubusercontent.com/47452191/173100457-641f7d3e-bf98-458e-aea5-4fe0ada3c31d.png)

**Comment by idoshamun** [comment]
Hi @RichestHumanAlive,

We were thinking quite a while about this kind of experience. 
It's not that trivial to pull off from a technical perspective, and we also want to make sure we don't affect the monetization channels of the publishers. This requires further investigation, and I hope we have time to do it in the upcoming months but can't promise anything, unfortunately. Thanks for your time and mock-ups 😁 

Cool username btw. 😉

**Reply by sonna-io** [reply]
@idoshamun Thanks a lot! 
This is absolutely understandable. 
Can't wait to see the upcoming versions 😊

Keep bringing this project up to stars, King.
